frontend/apps/app/app/(app)/app/organizations/[organizationId]/settings/general/DeleteConfirmationModal.module.css

```diff
@@ -0,0 +1,30 @@
+/* Modal styles */
+
+.confirmInputContainer {
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-4);
+  margin-top: var(--spacing-4);
+}
+
+.confirmLabel {
+  font-size: var(--font-size-3);
+  color: var(--global-mute-text);
+  margin: 0;
+}
+
+.organizationName {
+  font-weight: 500;
+  color: var(--global-foreground);
+}
+
+.confirmInput {
+  width: 100%;
+}
+
+.divider {
+  width: 100%;
+  height: 1px;
+  background-color: var(--global-border);
+  margin-top: var(--spacing-4);
+}
```

Comment:
There are some areas where the design (devider) is not applied properly due to the modal CSS, but I will address this later!

![ss 3167](https://github.com/user-attachments/assets/4ce604b9-1797-4101-b256-71bc8d77f1ef)